### ISSUE TITLE
A11y : Cannot tab into “Help” button next to Access field

### ISSUE THREAD

**Issue by yaelleC** [grafana/grafana#46471]
**Steps to reproduce**:
In the datasource creation page, although we see the info icon, it is not accessible via keyboard.
P1: Non blocking to set up the datasource but degraded experience

**Actual Result**:
Icon not focussable - help text not accessible

**Expected Result**
We should be able to tab into the icon and it would display the help text

**Relevant WCAG Criteria:**
[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
